test.js

```diff
@@ -100,3 +100,18 @@ test('can inject spans with a custom class prefix', function (t) {
     '<span class="c1" aria-hidden="true">f</span><span class="c2" aria-hidden="true">o</span><span class="c3" aria-hidden="true">o</span>'
   )
 })
+
+
+test('should support regex to split instead of letters.', function (t) {
+  t.plan(2)
+  const element = createElement(
+    '<span>আদর্শ লিপি</span>'
+  );
+  var bengaliUnitRegex = /([অ-হ](্[অ-হ])+|[অ-হ]্|[অ-হ])[া-ৌ]*ঁ*|ৎ| /g
+  charming(element, {}, bengaliUnitRegex);
+  t.equal(element.getAttribute('aria-label'), null)
+  t.equal(
+    element.innerHTML,
+    '<span aria-label="আদর্শ লিপি"><span class="char1" aria-hidden="true">আ</span><span class="char2" aria-hidden="true">দ</span><span class="char3" aria-hidden="true">র্শ</span><span class="char4" aria-hidden="true"> </span><span class="char5" aria-hidden="true">লি</span><span class="char6" aria-hidden="true">পি</span></span>'
+  );
+});
```

Comment:
Suggest having a simplified test case with a simple regular expression

Reply:
Ah! It's simple, trust me, for Bengali, it's the simplest!

Reply:
@uroybd 

This test failed https://travis-ci.org/yuanqing/charming/jobs/340147531#L520. (There’s an issue in the Travis script which I will fix separately.)

Am still of the opinion that this regular expression with non-alphabetic characters and character sets is extremely complex, and will be quite difficult to maintain going forward.

This PR actually addresses issue #1; can the test be a super simple one of splitting on words? For example:

```js
const element = createElement('foo bar')
charming(element, {
  splitRegex: /(\s+)/,
  classPrefix: 'word'
})
t.equal(element.getAttribute('aria-label'), 'foo bar')
t.equal(
  element.innerHTML,
  '<span class="word1" aria-hidden="true">foo</span><span class="word2" aria-hidden="true"> </span><span class="word3" aria-hidden="true">bar</span>'
)
```